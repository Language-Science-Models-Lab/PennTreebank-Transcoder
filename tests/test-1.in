(S @(NP+VP (NP~ @(NP (DT @The) @(NPR (NNP @National) @(NNP @Association))) (PP @(IN @of) (NP~ @(NPR @(NNP @Manufacturers))))) @(VP @(VBD+PP @(VBD @settled) (PP @(IN @on) (NP~ @(NP (DT @the) @(NPR+NP (NPR @(NNP @Hoosier)) @(NP @(NN @capital)))) (PP @(IN @of) (NP~ @(NPR @(NNP @Indianapolis))))))) (PP @(IN @for) (NP~ (PRP$ @its) @(NN+NN+NN (NN @fall) @(NN+NN (NN @board) @(NN @meeting))))))) (. @.))
