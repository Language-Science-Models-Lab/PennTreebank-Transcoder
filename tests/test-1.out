[[[The [National Association]] [of Manufacturers]] [[settled [on [[the [Hoosier capital]] [of Indianapolis]]]] [for [its [fall [board meeting]]]]]]:
